migration-engine/connectors/sql-migration-connector/src/sql_renderer/mysql_renderer.rs

```diff
@@ -280,7 +280,7 @@ impl SqlRenderer for MysqlFlavour {
 
         Ok(format!(
             "CREATE TABLE {} (\n{columns}{indexes}{primary_key}\n) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci",
-            table_name = self.quote_with_schema(table.name()),
+            table_name = self.quote(table.name()),
```

Comment:
/!\

Reply:
todo: create an issue for this (one for mysql, one for sqlite)

Reply:
and one for postgres to see if we can leave the schema name implicit in all cases

Reply:
The issues are created in the migrations-team repo